include/rle_data.hpp

```diff
@@ -326,7 +326,7 @@ namespace Gamera {
 	m_pos--;
 	if (!check_chunk()) {
 	  if (m_i != m_vec->m_data[m_chunk].begin()) {
-	    iterator prev_i = prev(m_i);
+	    iterator prev_i = std::prev(m_i);
```

Comment:
There were convenience functions `prev` and `next` defined at the [top in this file](https://github.com/jannschu/gamera/blob/bf8e1a20c848f1412148544a2786b7dc545e472d/include/rle_data.hpp#L155), which do the same as `std::prev` and `std::next`, I think.